stable/rbac-manager/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* robscott sudermanjr
```

Comment:
Do the usernames work with the `@` prefix?

Reply:
no, I don't think so